### PR TITLE
Bundle maplibre assets

### PR DESCRIPTION
**Pull request by cristineguadelupe**

### CODE REVIEW

**Comment by cristineguadelupe** [comment]
I'll be doing MapCell soon, as we're migrating everything to React

File: assets/maplibre/package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "dev": "esbuild main.js --bundle --minify --outfile=../../lib/assets/maplibre/build/main.js --format=esm --loader:.js=jsx --watch"
+  },
+  "dependencies": {
+    "@maplibre/maplibre-gl-geocoder": "^1.5.0",
+    "maplibre-gl": "^3.6.2",
+    "maplibre-gl-vector-text-protocol": "github:jimmyrocks/maplibre-gl-vector-text-protocol"
+  },
+  "devDependencies": {
+    "@types/events": "^3.0.3",
+    "@types/node": "^20.11.2",
```
Comment (jonatanklosko) [inline assets/maplibre/package.json:14]: Are these relevant without typescript?

**Review by jonatanklosko — APPROVED** [review]

File: assets/maplibre/package.json (hunk 1 above)
Comment (cristineguadelupe) [reply assets/maplibre/package.json:14]: The Geocoder plugin depends on them but it's not listing them correctly. They already have an open issue about this.

Comment (jonatanklosko) [reply assets/maplibre/package.json:14]: Makes sense!

**Pull request merged**